contrib/valgrind/valgrind_xlio.supp

```diff
@@ -90,21 +90,23 @@
    fun:_ZN15neigh_table_mgrC1Ev
 }
 {
-   rdma_create_event_channel_fp_leak
+   rdma_create_event_channel
+   Memcheck:Leak
+   obj:/*librdmacm.so*
+   ...
+   fun:rdma_create_event_channel
+   fun:create_rdma_channel
+   fun:_ZN15neigh_table_mgrC1Ev
+}
+{
+   rdma_create_event_channel
    Memcheck:Leak
    match-leak-kinds: possible
    fun:calloc
-   obj:/usr/lib/x86_64-linux-gnu/librdmacm.so.1.3.56.0
-   obj:/usr/lib/x86_64-linux-gnu/librdmacm.so.1.3.56.0
-   obj:/usr/lib/x86_64-linux-gnu/librdmacm.so.1.3.56.0
+   obj:/*librdmacm.so*
+   ...
    fun:rdma_create_event_channel
-   fun:_ZN15neigh_table_mgr19create_rdma_channelEv
    fun:UnknownInlinedFun
```

Comment:
compiling with clang "fun:_ZN15neigh_table_mgrC1Ev" from line 99 was inlined - ruining the specific stack suppression.
so I've also replaced the previous suppression with two suppressions: one for _ZN15neigh_table_mgr19create_rdma_channelEv and one for inlined func